demo/code.py

```diff
@@ -0,0 +1,15 @@
+import time
+import board
+import neopixel
+
+# Orange is modified here to make it more distinct from yellow
+# Remember to update the colours in src/code.py if you change them here
+corners = [0x00ff00, 0xff3300, 0xff00ff, 0xffff00]
+
+PIXELS = neopixel.NeoPixel(board.NEOPIXEL, 4)
+PIXELS.brightness = 0.02  # The LEDs are very bright
+
+# Preview corner colours
+PIXELS[:] = corners
+while True:
+    time.sleep(1)
```

Comment:
Question: Is this file run at any point? `src/code.py` is the entry point and I can't see this file referenced elsewhere?

Reply:
Forgot part of the readme. The demo folder is an alternative (and separate) firmware to allow for testing the corner colours.

Reply:
afbcf5d57f3a86c98ac6ab8ead619c77db20aa1f